Documentation/AEPEdgeIdentity.md

```diff
@@ -0,0 +1,137 @@
+# Getting Started
+
+# Android Test App
```

Comment:
L1 -> L3 Can we merge these two into `Getting started with the test app`

Reply:
Merged to suggested header!